examples/Cargo.toml

```diff
@@ -77,3 +78,6 @@ path = "native/native-server.rs"
 name = "esp32-server"
 path = "esp32/esp32-server.rs"
 
+[[bin]]
+name = "esp32-server-with-cred"
```

Comment:
Super naive question: if you can have a server with credentials, do we still need to have one without them?

Reply:
not ultimately, but because I'm trying to release the flasher iteratively I can't eliminate the other way of building the server until the end